src/backports/zoneinfo/__init__.pyi

```diff
@@ -30,6 +30,9 @@ class ZoneInfo(tzinfo):
     ) -> _T: ...
     @classmethod
     def clear_cache(cls, *, only_keys: Iterable[str] = ...) -> None: ...
+    def tzname(self, __dt: datetime | None) -> str | None: ...
+    def utcoffset(self, __dt: datetime | None) -> timedelta | None: ...
```

Comment:
I continue to request that this be merged as-is, because the purpose of a backport is to optimize compatibility with the released upstream version, not to pursue incompatible improvements. Obviously you’re welcome to continue pursuing those improvements upstream, and then we can backport them here in the future. But there’s no reason to block this current backport on that.